hashes/Cargo.toml

```diff
@@ -3,10 +3,10 @@ name = "bitcoin_hashes"
 version = "0.11.0"
 authors = ["Andrew Poelstra <apoelstra@wpsoftware.net>"]
 license = "CC0-1.0"
-description = "Hash functions used by rust-bitcoin which support rustc 1.41.1"
-homepage = "https://github.com/rust-bitcoin/bitcoin_hashes/"
 repository = "https://github.com/rust-bitcoin/bitcoin_hashes/"
 documentation = "https://docs.rs/bitcoin_hashes/"
+description = "Hash functions used by the rust-bitcoin eccosystem"
+categories = ["Algorithms"]
```

Comment:
IIUC categories are not capitalized.

Reply:
Heads up that this is still not addressed.

Reply:
oh man, thanks. My bad.